Source/AliveLibAO/Abe.cpp

```diff
@@ -3064,6 +3066,8 @@ __int16 Abe::VTakeDamage_4214E0(BaseGameObject* pFrom)
 
     field_11C_regen_health_timer = gnFrameCount_507670 + 900;
 
+    auto oldHpGtZero = field_100_health > FP_FromInteger(0);
```

Comment:
surely just store const FP oldHP = field_100_health ; and check > 0 at the end? :P

Reply:
it's not const on purpose :shook:

Reply:
I mean about checking > 0 in the local var - that is an odd compiler/ida-ism